### DISCUSSION TITLE
Statement of Owner Equity

### DISCUSSION THREAD

**Discussion by einhverfr** [ledgersmb/LedgerSMB#5831] [Ideas]
This, and the statement of cash flows, are the only financial statements we don;t currently support.

https://sourceforge.net/p/ledger-smb/feature-requests/99/

**Comment by ehuelsmann** [comment]
Actually, I had a potential customer ask me about this type of report the other week!

**Comment by einhverfr** [comment]
I dont think it would be too hard to do in the current setup.  It could be done in 1.5 without any further work since it is basically a differential balance sheet.
